wca-integration/src/main/java/eu/itesla_project/wca/WCAImpl.java

```diff
@@ -415,6 +423,8 @@ public WCACluster after(Path workingDir, ExecutionReport report) throws IOExcept
                             network.getStateManager().setWorkingState(preventiveStateId);
                             AmplUtil.fillMapper(mapper, network); // because action can create a new bus
                             new AmplNetworkWriter(network, dataSource, 0, preventiveActionNum, true, mapper, DOMAINS_AMPL_EXPORT_CONFIG).write();
+                            if (config.isExportStates())
+                                new WCAStateExporter(network, workingDir, 0, preventiveActionNum).export();
```

Comment:
Add braces to respect java coding guideline even if there is only one statement.